Multisig/UI/Settings/OwnerKeyManagement/OwnerKeyDetailsViewController/OwnerKeyDetailsViewController.swift

```diff
@@ -248,19 +248,7 @@ class OwnerKeyDetailsViewController: UITableViewController, WebConnectionObserve
         switch item {
         case Section.Backedup.backedup:
             return tableView.backupKeyCell(indexPath: indexPath) { [weak self] in
-                guard let self = self else { return }
```

Comment:
ok this has to be implemented in the new flows so that the authentication is there for the backup to start. I'll push to this branch to fix this